internal/args/parse.go

```diff
@@ -77,6 +77,8 @@ type Config struct {
 	// This should be parsed but not exposed
 	Filter filterArgument `short:"f" hidden:"1"`
 
+	Exclude []string `placeholder:"PATTERN" help:"Exclude files matching this pattern"`
```

Comment:
Could you please add this to README.md as well?

Reply:
Done